pkg/network/tracer/offsetguess.go

```diff
@@ -796,19 +803,33 @@ func newEventGenerator(ipv6 bool) (*eventGenerator, error) {
 	return eg, nil
 }
 
-func getUDP6Conn(ipv6 bool) (*net.UDPConn, error) {
+func getUDP6Conn(ipv6 bool) (c *net.UDPConn, err error) {
 	if !ipv6 {
 		return nil, nil
 	}
 
-	linkLocal, err := getIPv6LinkLocalAddress()
-	if err != nil {
-		// TODO: Find a offset guessing method that doesn't need an available IPv6 interface
-		log.Debugf("unable to find ipv6 device for udp6 flow offset guessing. unconnected udp6 flows won't be traced: %s", err)
-		return nil, nil
-	}
+	// it is necessary to run the following code in
```

Comment:
out of curiosity do you know why the host & root NS would have IPv6 enabled but the container NS would not?

Reply:
I think you have to explicitly enable it. With `docker` it is not enabled by default.